tests/integration/test_fim/test_files/test_ambiguous_confs/data/test_cases/cases_whodata_works_over_realtime.yaml

```diff
@@ -0,0 +1,19 @@
+- name: whodata_thread_started_with_realtime_enabled
+  description: Check Whodata thread is enabled and is used with realtime enabled on same tag
+  configuration_parameters:
+    WHODATA: 'yes'
+    REALTIME: 'yes'
+    TEST_DIR: !!python/object/apply:os.path.join [/, test_dir]
+  metadata:
+    folder_to_monitor: !!python/object/apply:os.path.join [/, test_dir]
+    file_to_monitor: !!python/object/apply:os.path.join [/, test_dir, test_file.log]
+
+- name: whodata_thread_started_with_realtime_disabled
+  description: Check Whodata thread is enabled and is used with realtime disabled on same tag
+  configuration_parameters:
+    WHODATA: 'yes'
+    REALTIME: 'no'
+    TEST_DIR: !!python/object/apply:os.path.join [/, test_dir]
+  metadata:
+    folder_to_monitor: !!python/object/apply:os.path.join [/, test_dir]
+    file_to_monitor: !!python/object/apply:os.path.join [/, test_dir, test_file.log]
```

Comment:
Missing line a the end of the file. Check all files with this problem.

Reply:
Done